“settings.DATABASES is improperly configured”
./manage.py --settings=nameofproject.settings runserver/syncdb